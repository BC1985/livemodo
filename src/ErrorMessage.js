import React from "react";
const errorStyle = {
  color: "red"
};
function ErrorMessage() {
  return <div style={errorStyle}>Error</div>;
}

export default ErrorMessage;
