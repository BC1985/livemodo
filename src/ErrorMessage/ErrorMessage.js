import React from "react";
const errorStyle = {
  textAlign: "center",
  color: "white"
};
function ErrorMessage() {
  return <div style={errorStyle}>Something went wrong, please try again.</div>;
}

export default ErrorMessage;
