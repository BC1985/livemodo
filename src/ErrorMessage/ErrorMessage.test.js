import React from "react";
import ReactDOM from "react-dom";
import ErrorMessage from "../ErrorMessage/ErrorMessage";
import renderer from "react-test-renderer";

it("renders without crashing", () => {
  const div = document.createElement("div");
  ReactDOM.render(<ErrorMessage />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it("renders the UI as exptected", () => {
  const tree = renderer.create(<ErrorMessage />).toJSON();
  expect(tree).toMatchSnapshot();
});
