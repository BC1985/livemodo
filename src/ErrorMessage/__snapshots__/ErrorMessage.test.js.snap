// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the UI as exptected 1`] = `
<div
  style={
    Object {
      "color": "white",
      "textAlign": "center",
    }
  }
>
  Something went wrong, please try again.
</div>
`;
